### PR TITLE
Resolving module docs inconsistencies

### LINKED ISSUE #74: Some module docs inconsistencies
##### SUMMARY
In https://github.com/ansible-collections/amazon.aws/pull/72 we identified several places where module docs could have better or more consistent style, especially around capitalization.

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME
ec2_eni_info.py
aws_s3.py

It would be good if any patch fixing this also runs https://github.com/ansible-network/collection_prep/blob/master/add_docs.py to generate new docs.

### PR DESCRIPTION
**Pull request by mandar242**
##### SUMMARY

Resolving module docs inconsistencies.
Resolves #74 
Related to #72 

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: docs/amazon.aws.aws_az_info_module.rst [hunk 1]
```diff
@@ -55,7 +55,7 @@ Parameters
                 <td>
                 </td>
                 <td>
-                        <div>AWS access key. If not set then the value of the AWS_ACCESS_KEY_ID, AWS_ACCESS_KEY or EC2_ACCESS_KEY environment variable is used.</div>
+                        <div><code>AWS access key</code>. If not set then the value of the AWS_ACCESS_KEY_ID, AWS_ACCESS_KEY or EC2_ACCESS_KEY environment variable is used.</div>
```
Comment (tremble) [inline docs/amazon.aws.aws_az_info_module.rst:58]: I don't see any markup in the aws.py fragment that would have generated these code blocks, were they manually added?

**Review by tremble — CHANGES_REQUESTED** [review]
Hi @mandar242,

Thanks for taking the time to open this PR.  The rst files should all be autogenerated from the docs variables in the python code.  (Some of the docs live in the fragments)  It looks like you've manually added code blocks around the start of the descriptions for the credential parameters.  This will be automatically removed the next time the docs generation is run.

You'll need to wrap the corresponding entries in `C(.....)`(plugins/doc_fragments/aws.py):

```
  aws_secret_key:
    description:
      - C(AWS secret key). If not set then the value of the C(AWS_SECRET_ACCESS_KEY), C(AWS_SECRET_KEY), or C(EC2_SECRET_KEY) environment variable is used.
      - If I(profile) is set this parameter is ignored.
      - Passing the I(aws_secret_key) and I(profile) options at the same time has
```

File: docs/amazon.aws.aws_az_info_module.rst (hunk 1 above)
Comment (mandar242) [reply docs/amazon.aws.aws_az_info_module.rst:58]: Thank you for your valuable feedback @tremble.

**Review by tremble — APPROVED** [review]

File: plugins/doc_fragments/aws.py [hunk 2]
```diff
@@ -21,30 +21,30 @@ class ModuleDocFragment(object):
...
-      - Url to use to connect to EC2 or your Eucalyptus cloud (by default the module will use EC2 endpoints).
+      - URL to use to connect to EC2 or your Eucalyptus cloud (by default the module will use EC2 endpoints).
         Ignored for modules where region is required. Must be specified for all other modules if region is not used.
         If not set then the value of the EC2_URL environment variable, if any, is used.
     type: str
     aliases: [ aws_endpoint_url, endpoint_url ]
   aws_secret_key:
     description:
-      - AWS secret key. If not set then the value of the AWS_SECRET_ACCESS_KEY, AWS_SECRET_KEY, or EC2_SECRET_KEY environment variable is used.
+      - C(AWS secret key). If not set then the value of the AWS_SECRET_ACCESS_KEY, AWS_SECRET_KEY, or EC2_SECRET_KEY environment variable is used.
```
Comment (tremble) [inline plugins/doc_fragments/aws.py:31]:
```suggestion
      - C(AWS secret key). If not set then the value of the C(AWS_SECRET_ACCESS_KEY), C(AWS_SECRET_KEY), or C(EC2_SECRET_KEY) environment variable is used.
```

File: plugins/doc_fragments/aws.py [hunk 3]
```diff
@@ -21,30 +21,30 @@ class ModuleDocFragment(object):
...
       - If I(profile) is set this parameter is ignored.
       - Passing the I(aws_secret_key) and I(profile) options at the same time has been deprecated
         and the options will be made mutually exclusive after 2022-06-01.
     type: str
     aliases: [ ec2_secret_key, secret_key ]
   aws_access_key:
     description:
-      - AWS access key. If not set then the value of the AWS_ACCESS_KEY_ID, AWS_ACCESS_KEY or EC2_ACCESS_KEY environment variable is used.
+      - C(AWS access key). If not set then the value of the AWS_ACCESS_KEY_ID, AWS_ACCESS_KEY or EC2_ACCESS_KEY environment variable is used.
```
Comment (tremble) [inline plugins/doc_fragments/aws.py:39]:
```suggestion
      - C(AWS access key). If not set then the value of the C(AWS_ACCESS_KEY_ID), C(AWS_ACCESS_KEY) or C(EC2_ACCESS_KEY) environment variable is used.
```

File: plugins/doc_fragments/aws.py [hunk 4]
```diff
@@ -21,30 +21,30 @@ class ModuleDocFragment(object):
...
       - If I(profile) is set this parameter is ignored.
       - Passing the I(aws_access_key) and I(profile) options at the same time has been deprecated
         and the options will be made mutually exclusive after 2022-06-01.
     type: str
     aliases: [ ec2_access_key, access_key ]
   security_token:
     description:
-      - AWS STS security token. If not set then the value of the AWS_SECURITY_TOKEN or EC2_SECURITY_TOKEN environment variable is used.
+      - C(AWS STS security token). If not set then the value of the AWS_SECURITY_TOKEN or EC2_SECURITY_TOKEN environment variable is used.
```
Comment (tremble) [inline plugins/doc_fragments/aws.py:47]:
```suggestion
      - C(AWS STS security token). If not set then the value of the C(AWS_SECURITY_TOKEN) or C(EC2_SECURITY_TOKEN) environment variable is used.
```

File: plugins/modules/aws_s3.py [hunk 5]
```diff
@@ -127,28 +127,28 @@
...
   src:
     description:
-      - The source file path when performing a PUT operation.
-      - Either I(content), I(content_base64) or I(src) must be specified for a PUT operation. Ignored otherwise.
+      - The source file path when performing a C(PUT) operation.
+      - Either I(content), I(content_base64) or I(src) must be specified for a C(PUT) operation. Ignored otherwise.
     type: path
   content:
     description:
       - The content to PUT into an object.
```
Comment (tremble) [inline plugins/modules/aws_s3.py:135]:
```suggestion
      - The content to C(PUT) into an object.
```

File: plugins/modules/aws_s3.py [hunk 6]
```diff
@@ -127,28 +127,28 @@
...
       - The parameter value will be treated as a string and converted to UTF-8 before sending it to S3.
         To send binary data, use the I(content_base64) parameter instead.
-      - Either I(content), I(content_base64) or I(src) must be specified for a PUT operation. Ignored otherwise.
+      - Either I(content), I(content_base64) or I(src) must be specified for a C(PUT) operation. Ignored otherwise.
     version_added: "1.3.0"
     type: str
   content_base64:
     description:
       - The base64-encoded binary data to PUT into an object.
```
Comment (tremble) [inline plugins/modules/aws_s3.py:143]:
```suggestion
      - The base64-encoded binary data to C(PUT) into an object.
```

**Review by tremble — COMMENTED** [review]
Looking at #72 it's probably worth wrapping the env variable names too.

**Review by tremble — APPROVED** [review]

**Pull request merged**